guides/advanced_guides/known_limitations.md

```diff
@@ -0,0 +1,27 @@
+# Known limitations
+
+MeiliSearch has known limitations, some of which may be addressed in future versions
+
+## Design limitations
+
+### Database size
+
+MeiliSearch uses two database: one for storage and the second one for updates.
+Default database maximum size is 100GiB.
+MeiliSearch uses LMDB under the hood as the storage engine. On launch, LMDB need to know the size that it can allocate on disk. This space will be reserved on disk for LMDB, thus MeiliSearch. This space will also be allocated as virtual memory, which is different from the real memory. You can read more [about storage](/resources/about_storage.md).
+You can modify this default limit using the options `--max-mdb-size` & `--max-udb-size` as described in the [configuration guide](/guides/advanced_guides/configuration.md#max-mdb-size).
+
+### Number of indexes
+
+You can create up to 200 indexes in MeiliSearch. When using LMDB, MeiliSearch have to specify a number of databases that must stay low for performance reasons.
```

Comment:
```suggestion
You can create up to 200 indexes in MeiliSearch. When using LMDB, MeiliSearch has to specify a number of databases that must stay low for performance reasons.
```

Reply:
>  When using LMDB, MeiliSearch has to specify a number of databases that must stay low for performance reasons. 

I do not understand this sentence :s

Reply:
Me neither. Does it mean to impose a limit of X GiB on the size of these databases or on the number of indexes?

Reply:
I re-wrote the whole paragraph